### PR TITLE
Fix: Concordium integration tests

### PR DESCRIPTION
**Pull request by MartinSchmidt**

### CODE REVIEW

File: src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/README.md [hunk 1]
```diff
@@ -4,13 +4,12 @@ To be able to run the integration test with Concordium,
...
 
 This can be achieved using the included [docker-compose](../ProjectOrigin.VerifiableEventStore.Tests/docker-compose.yaml).
-One forking the repo, one must get a token from GitHub to be able to join runners.
 
 ## Running a node
 1. Make a .env file containing the following environment variables:
 
 ```sh
-GITHUB_RUNNER_TOKEN=#YOUR_GITHUB_RUNNER_TOKEN
+GITHUB_RUNNER_PAT=${YOUR_GITHUB_ORGANIZATION_PAT}
```
Comment (wisbech) [inline src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/README.md:12]:
```suggestion
GITHUB_RUNNER_PAT=${YOUR_GITHUB_ORGANIZATION_PATH}
```

File: src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/docker-compose.yaml [hunk 2]
```diff
@@ -6,22 +6,23 @@ services:
       context: .
       dockerfile: github-runner.dockerfile
       args:
-        TOKEN: ${GITHUB_RUNNER_TOKEN}
-        REPOSITORY: https://github.com/project-origin/registry
+        PAT: ${GITHUB_RUNNER_PAT}
```
Comment (wisbech) [inline src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/docker-compose.yaml:9]:
```suggestion
        PAT: ${GITHUB_RUNNER_PATH}
```

File: src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/github-runner.dockerfile [hunk 3]
```diff
@@ -2,9 +2,9 @@
 FROM ubuntu:20.04
 
 # Github runner version https://github.com/actions/runner/releases
-ARG VERSION="2.298.2"
-ARG TOKEN
-ARG REPOSITORY="https://github.com/project-origin/registry"
+ARG VERSION="2.303.0"
+ARG PAT
```
Comment (wisbech) [inline src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/github-runner.dockerfile:6]:
```suggestion
ARG PATH
```

File: src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/github-runner.dockerfile [hunk 4]
```diff
@@ -36,6 +36,6 @@ RUN curl -o runner.tar.gz -L https://github.com/actions/runner/releases/download
 USER runner
 
 # Configure runner
-RUN ./config.sh --unattended --url ${REPOSITORY} --token ${TOKEN} --replace --name ${NAME} --labels ${LABELS}
+RUN ./config.sh --unattended --url ${REPOSITORY} --pat ${PAT} --replace --name ${NAME} --labels ${LABELS}
```
Comment (wisbech) [inline src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/github-runner.dockerfile:39]:
```suggestion
RUN ./config.sh --unattended --url ${REPOSITORY} --pat ${PATH} --replace --name ${NAME} --labels ${LABELS}
```

File: src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/README.md [hunk 5]
```diff
@@ -4,13 +4,12 @@ To be able to run the integration test with Concordium,
...
 
 This can be achieved using the included [docker-compose](../ProjectOrigin.VerifiableEventStore.Tests/docker-compose.yaml).
-One forking the repo, one must get a token from GitHub to be able to join runners.
 
 ## Running a node
 1. Make a .env file containing the following environment variables:
 
 ```sh
-GITHUB_RUNNER_TOKEN=#YOUR_GITHUB_RUNNER_TOKEN
+GITHUB_RUNNER_PAT=${YOUR_GITHUB_ORGANIZATION_PATH}
```
Comment (wisbech) [inline src/ProjectOrigin.VerifiableEventStore.ConcordiumIntegrationTests/README.md:12]:
```suggestion
GITHUB_RUNNER_PATH=${YOUR_GITHUB_ORGANIZATION_PATH}
```

**Review by wisbech — APPROVED** [review]
Looks great - have committed suggestion on spelling - completely non-functional

**Comment by MartinSchmidt** [comment]
@wisbech the reason it was PAT is an abbreviation for Personal Access Token, and not for a path in the filesystem.

**Comment by wisbech** [comment]
> @wisbech the reason it was PAT is an abbreviation for Personal Access Token, and not for a path in the filesystem.

My bad then - I had no idea of this abbreviation, but it makes sense now. Should we change this back ?

**Pull request merged**